scripts/tests/test_obfi.py

```diff
@@ -3,41 +3,43 @@
 import socket
 import sys
 import time
-import urllib.request
 from pathlib import Path
 from types import MappingProxyType
 
 import pytest
+import requests
 
 os.environ['SEED_PATH'] = 'must be truthy for obfi/decode_ip scripts to run'
 from ..obfi import hide, mktable, reveal, shownames
 
 
-def mock_urlopen(*args, **kwargs):
-    """Mock for urllib.request.urlopen to always return seed=1234."""
+def mock_get(*args, **kwargs):
+    """Mock for requests.get to always return seed=1234."""
 
-    class MockRead:
-        def read(self):
-            return b"seed=1234"
+    class MockGet:
+        text = b"seed=1234"
```

Comment:
Since it's bytes, this should now be `content`
```suggestion
        content = b"seed=1234"
```